### ISSUE TITLE
[Prisa Media] Domain limitations in FPS

### ISSUE THREAD

**Issue by nlozanoarguelles** [WICG/first-party-sets#153]
Hello,
I am the Data Director of [Prisa Media](https://www.prisa.com/), one of the largest media groups providing Spanish-speaking content. Our group consists of over 20 individual brands, each with its own specific domain. **Our primary goal is to build a seamless cross-brand experience for our users**, allowing them to effortlessly access and engage with content across our various brands.
However, the current proposal to enable first party cookies across multiple domains through the First Party Sets, which limits this feature to a maximum of three domains, poses a significant challenge to achieving this seamless cross-brand experience. With the deprecation of third-party cookies, tracking user preferences and facilitating cross-domain logged-in experiences will be significantly hindered, as **over 90% of our traffic comes from anonymous users**.
Restricting the number of domains that can share first-party cookies to three favors organizations that either have one large brand or have adopted a subdomain strategy. In our case, we have distinct vertical brands focused on different topics and formats (news, audio, video), and we prioritize brand recognition linked to content over the overarching Prisa Media brand message.
We propose that the feature should not limit the number of domains that can share first-party cookies. Instead, the focus should be on enabling a seamless cross-brand experience by validating and exemplifying how the feature will be used. Consideration could be given to limiting the names of cookies that are allowed to maintain value across domains, rather than restricting the number of domains themselves.
**By removing the three-domain limit, First Party Sets would empower organizations like ours to provide a truly seamless user experience across multiple domains, benefitting both users and content providers**.
We appreciate your attention to this matter and hope you will consider our proposal in support of seamless cross-brand experiences.
Best regards,
The Prisa Media Team

**Comment by brownwolf1355** [comment]
I agree with @nlozanoarguelles. There are many media groups that publish under numerous brands (much greater than 3), e.g., Gannette, Meredith, AdvanceLocal, American Cities Business Journal, etc. It seems like organizations with a large number of associatedSites could be highlighted and the community as a whole could review the validity of the FPS (e.g., confirm that it isn't collusion among a group of sites). This does raise the issue of how these kinds of circumstances are adjudicated if a fair and equitable manner.

**Comment by krgovind** [comment]
Thanks for sharing your feedback, @nlozanoarguelles. I have re-posted your comment to #93, since we are tracking feedback on the numeric limit in that issue.

> Consideration could be given to limiting the names of cookies that are allowed to maintain value across domains, rather than restricting the number of domains themselves.

Regarding the restriction on the number of domains - note that First-Party Sets did not initially have the numeric limit; but we changed the proposal to have a limit (among other changes) based on community feedback. We explained the rationale for the changes [here](https://developer.chrome.com/docs/privacy-sandbox/first-party-sets-evolution). 

Could you clarify how restrictions on the names of cookies might help? I wonder if you're asking about limiting the amount of informational entropy shared across domains, as being discussed in #111?

**Comment by Nathaniel-Francis** [comment]
Hello, I'm the Revenue Operations Director at Bauer Media and i strongly agree with @nlozanoarguelles.  We also have a portfolio of sites and the three domain limit on first party sets would cause the same issues for us as outlined above
